library/src/test/kotlin/com/chuckerteam/chucker/api/ChuckerInterceptorSkipRequestTest.kt

```diff
@@ -0,0 +1,352 @@
+package com.chuckerteam.chucker.api
+
+import com.chuckerteam.chucker.util.ChuckerInterceptorDelegate
+import com.chuckerteam.chucker.util.ClientFactory
+import com.chuckerteam.chucker.util.NoLoggerRule
+import com.chuckerteam.chucker.util.readByteStringBody
+import com.google.common.truth.Truth.assertThat
+import io.mockk.every
+import io.mockk.mockk
+import okhttp3.Headers
+import okhttp3.HttpUrl
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.Rule
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.io.TempDir
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+import java.io.File
+
+@ExtendWith(NoLoggerRule::class)
+internal class ChuckerInterceptorSkipRequestTest {
+    @get:Rule
+    val server = MockWebServer()
+
+    @TempDir
+    lateinit var tempDir: File
+
+    @ParameterizedTest
+    @EnumSource(value = ClientFactory::class)
+    fun `chucker processes all requests when no skipEndpoints are provided`(factory: ClientFactory) {
+        val chuckerInterceptorWithoutSkipping =
+            ChuckerInterceptorDelegate(
+                cacheDirectoryProvider = { tempDir },
+            )
+        val client = factory.create(chuckerInterceptorWithoutSkipping)
+        executeRequestForPath(client, "/", "Response from /")
+        val transaction = chuckerInterceptorWithoutSkipping.expectTransaction()
+        assertThat(transaction.responseBody).isEqualTo("Response from /")
+        chuckerInterceptorWithoutSkipping.expectNoTransactions()
```

Comment:
Added `chuckerInterceptorWithoutSkipping.expectNoTransactions()` to ensure false positive checks.